projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.ts

```diff
@@ -55,7 +55,8 @@ export class AddToCartComponent implements OnInit, OnDestroy {
           if (
             product.stock &&
             product.stock.stockLevelStatus !== 'outOfStock' &&
-            product.stock.stockLevel > 0
+            product.stock.stockLevel > 0 &&
+            product.purchasable
```

Comment:
This is likely not available on the SOLR fields, I rather fix the sample data here, see https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/3044#issuecomment-547309643